.env

```diff
@@ -0,0 +1,6 @@
+###> symfony/framework-bundle ###
```

Comment:
even not installed with Flex, used this syntax so Flex is ready to identify those places when we will install it and update recipes